docs/project/plugins.sbt

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.6.0-SNAPSHOT"))
```

Comment:
I think travis isn't set up to run the documentation tests; otherwise we would've caught this.